docs/docs/recipes/configure-storage-providers/README.md

```diff
@@ -59,14 +59,14 @@ S3 Storage is a cloud storage service that offers object storage through a web s
 Example usage:
 
 ```
-pnpm logto db system set storageProvider '{"provider":"S3Storage","accessKeyId":"my-access-key-id","secretAccessKey": "my-secret-access-key","bucket":"logto","endpoint":"https://s3.us-east-2.amazonaws.com"}'
+pnpm logto db system set storageProvider '{"provider":"S3Storage","accessKeyId":"my-access-key-id","accessSecretKey": "my-secret-access-key","bucket":"logto","endpoint":"https://s3.us-east-2.amazonaws.com"}'
```

Comment:
should we update in versioned docs as well? since it's a fix

Reply:
updated